tests/model_inheritance/tests.py

```diff
@@ -238,8 +238,8 @@ def test_init_subclass(self):
         saved_kwargs = {}
 
         class A(models.Model):
-            def __init_subclass__(cls, **kwargs):
-                super().__init_subclass__()
+            def __init_subclass__(cls, cls_meta=None, **kwargs):
+                super().__init_subclass__(cls_meta=None)
```

Comment:
```suggestion
                super().__init_subclass__(cls_meta=cls_meta)
```
There may be a need for more tests to cover the issue above?